### PR TITLE
Language attributes

### PR DESCRIPTION
**Pull request by dashea**
In certain cases, Pango needs to know what language to render text in. The default language cannot be changed once the application has started, so work around that by explicitly setting language attributes on all labels instead.

### CODE REVIEW

File: pyanaconda/ui/gui/utils.py [hunk 1]
```diff
@@ -520,3 +520,70 @@ def find_first_child(parent, match_func):
...
+
+        watch_children(widget, callback, user_data)
+
+    def _remove_signal(container, widget, user_data):
+        callback, user_data = user_data
+
+        unwatch_children(widget, callback, user_data)
+
+    callback(widget, user_data)
+    if (isinstance(widget, Gtk.Container)):
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/utils.py]: The brackets ``()`` aren't required here.

File: pyanaconda/ui/gui/utils.py [hunk 2]
```diff
@@ -520,3 +520,70 @@ def find_first_child(parent, match_func):
...
+
+            _widget_watch_list[signal_key] = (add_signal, remove_signal)
+
+def unwatch_children(widget, callback, user_data=None):
+    """
+    Unregister a callback previously added with watch_children.
+
+    :param GtkWidget widget: the widget to unwatch
+    :param function callback: the callback that was previously added to the widget
+    :param user_data: The user_data that was previously added to the widget
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/utils.py]: You are using lower case ``the`` in the first two :param and you have ``The`` in the third case.

**Comment by jkonecny12** [comment]
Seems good to me except to those two small neat picks.

**Comment by dashea** [comment]
Fixed the capitalization, removed the C-style parens, and pushed.

**Pull request closed without merging**